source/draw/Image.ooc

```diff
@@ -11,42 +11,26 @@ use base
 use draw
 import Canvas
 
-CoordinateSystem: enum {
-	Default = 0x00
-	XRightward = 0x00
-	XLeftward = 0x01
-	YDownward = 0x00
-	YUpward = 0x02
-}
```

Comment:
Having the flipping here was too complicated and caused lots of direct contradictions in what to do.

For example: Moving an image from GPU to CPU may not flip because we don't know if it will be done using rendering or a direct copy. At the same time, we must flip when moving an image to the CPU because that is the final chance to apply flipping before the image is loaded using pointers with memory hacks where flipping is not possible.